cmds/request/wow/character.js

```diff
@@ -1,9 +1,9 @@
 #! /usr/bin/env node
 
 const yargs = require('yargs');
-const blizzard = require('blizzard.js').initialize({ apikey: process.env.BATTLENET_CLIENT_ID });
+const logger = require('../../../lib/logger');
```

Comment:
This tends to be the convention for node modules. With a properly structured application it's common to see relative path imports but this will be simplified when the logger is moved to its own package